src/main/kotlin/racingcar/domain/RidingCount.kt

```diff
@@ -0,0 +1,7 @@
+package racingcar.domain
+
+class RidingCount(private val _value: Int){
+
+    val value
+    get() = _value
+}
```

Comment:
```suggestion
class RidingCount(private val _value: Int) {

    val value
        get() = _value
}
```

Please format the code.